### ISSUE TITLE
Stack Problem & crash

### ISSUE THREAD

**Issue by Boyeen** [VolosR/bitCoinTTGO#1]
Hi,
Just FYI they seem to have made a change on Cryptonator that crashes your sketch. The API call no longer gets a direct JSON reply. They have added a DDOS browser check which returns quite a large file and overflows stack?  At least it appears that is what is happening.

**Comment by cryptostarfish** [comment]
Yes unfortunately the TTGO will connect to WiFi, begin drawing and then crash. Assuming this is the API issue, has anyone found a way to resolve this? It's a great project, perhaps Cryptonator is not the best answer?

**Comment by fiatbravodriver** [comment]
I also would like to know please.
